parmed/formats/pdb.py

```diff
@@ -736,6 +737,9 @@ def write(struct, dest, renumber=True, coordinates=None, altlocs='all',
         standard_resnames : bool, optional, default False
             If True, common aliases for various amino and nucleic acid residues
             will be converted into the PDB-standard values.
+        increase_tercount : bool, optional, default True
+            If True, the TER atom number field increased by one compared to
+            atom card preceding it.
```

Comment:
Add a comment here that `True` conforms to PDB standard.  That's the reason this was the default behavior before.